plugins/star-rating/tests.js

```diff
@@ -7,55 +7,255 @@ var API_KEY_ADMIN = "";
-			var params = {app_id:APP_ID, period:"reset"};
-			request
-			.get('/i/apps/reset?api_key='+API_KEY_ADMIN+"&args="+JSON.stringify(params))
-			.expect(200)
-			.end(function(err, res){
-				if (err) return done(err);
-				var ob = JSON.parse(res.text);
-				ob.should.have.property('result', 'Success');
-				setTimeout(done, 5000)
-			});
-		});
-	});
+    describe('Reset app', function() {
+        it('should reset data', function(done) {
+            var params = {
+                app_id: APP_ID,
+                period: "reset"
+            };
+            request.get('/i/apps/reset?api_key=' + API_KEY_ADMIN + "&args=" + JSON.stringify(params)).expect(200).end(function(err, res) {
+                if (err) return done(err);
+                var ob = JSON.parse(res.text);
+                ob.should.have.property('result', 'Success');
+                setTimeout(done, 1000);
+            });
+        });
+    });
+
+    describe('Get empty widget list after reset app', function() {
+        it('should return 200 and empty widget list', function(done) {
+            API_KEY_ADMIN = testUtils.get("API_KEY_ADMIN");
+            APP_ID = testUtils.get("APP_ID");
+            request.get('/o/feedback/widgets?app_id=' + APP_ID + '&api_key=' + API_KEY_ADMIN)
+            .expect(200)
+            .end(function(err, res) {
+                if (err) return done(err);
+                var ob = JSON.parse(res.text);
+                ob.length.should.eql(0);
+                setTimeout(done, 100);
+            });
+        });
+    });
+
+    describe('Get single widget after reset app with created widget\'s id', function() {
+        it('should return 404', function(done) {
+            API_KEY_ADMIN = testUtils.get("API_KEY_ADMIN");
+            APP_ID = testUtils.get("APP_ID");
+            WIDGET_ID = testUtils.get("WIDGET_ID");
+            request.get('/o/feedback/widget?app_id=' + APP_ID + '&api_key=' + API_KEY_ADMIN + '&widget_id=' + WIDGET_ID)
+            .expect(404)
+            .end(function(err, res) {
+                if (err) return done(err);
+                setTimeout(done, 100);
+            });
+        });
+    });
 
+    describe('Get widgets by array parameter after reset app with created widget\'s id', function() {
+        it('should return widgets array', function(done) {
+            API_KEY_ADMIN = testUtils.get('API_KEY_ADMIN');
+            APP_ID = testUtils.get("APP_ID");
+            WIDGET_ID = testUtils.get('WIDGET_ID');
+            var array4query = [WIDGET_ID];
+            request.get('/o/feedback/multiple-widgets-by-id?app_id=' + APP_ID + '&api_key=' + API_KEY_ADMIN + '&widgets=' + JSON.stringify(array4query))
+            .expect(200)
+            .end(function(err, res) {
+                if (err) return done(err);
+                var ob = JSON.parse(res.text);
+                ob.length.should.eql(0);
+                setTimeout(done, 100);
+            });
+        });
+    });
```

Comment:
Here we should also verify feedback data, that it is empty and was removed on reset